.github/workflows/release-testflight.yml

```
@@ -0,0 +1,306 @@
+              exit 0
+            fi
+            sleep 5
+          done
+          echo "⚠️ Timed out waiting for summary."
+          echo "summary_found=false" >> $GITHUB_OUTPUT
+          exit 1
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)